src/Data/Bit/Immutable.hs

```diff
@@ -680,10 +670,6 @@ unsafeNthTrueInWord l w = countTrailingZeros (pdep (1 `shiftL` (l - 1)) w)
 -- @since 0.1
 countBits :: U.Vector Bit -> Int
 countBits (BitVec _ 0 _)                      = 0
-#if UseLibGmp
-countBits (BitVec 0 len arr) | modWordSize len == 0 =
-  fromIntegral (mpnPopcount arr (divWordSize len))
```

Comment:
Could you please implement `countBits` as well?

Reply:
I managed to come up with a version that has about the same speed as the GMP version.